code/modules/clothing/clothing.dm

```diff
@@ -38,17 +40,8 @@
 	// THESE OVERRIDE THE HIDEHAIR FLAGS
 	var/dynamic_hair_suffix = ""//head > mask for head hair
 	var/dynamic_fhair_suffix = ""//mask > head for facial hair
-	var/greyscale = FALSE

```

Comment:
dude you removed all the stuff to make sure that armors reflect their material color